### DISCUSSION TITLE
Inventory spares management

### DISCUSSION THREAD

**Discussion by AndMrzv** [netbox-community/netbox#10911] [Q&A]
So I want to keep track of spare inventory items I have on my shelves in a quantifiable way - let's say for example

- 70 units of "noname 40mm fan 4pin"
- 35 units of "standard 1u PSU 450-600w"

something I absolutely have to have counted but do not care enough to store dozens of proper instances with their own serial numbers, roles, statuses and everything. Things just lying in boxes so my guys could routinely use them to swap similar ones when it's due. Of course changing the total number when they take one out or restock.

Any ideas on how to do that properly within Netbox? Am I trying to force on it something it's not meant for?

Google gives me 2 inventory plugins but they seem to solve tasks different from mine.

**Comment by mtinberg** [comment]
Yeah, I don't think Netbox is quite the right tool for that job, if you were tracking serialized inventory then having Device records with status Inventory and Locations for each shelving unit/box could be done, but if you are just tracking counts than another asset/inventory tracking tool is probably a better fit.  I don't know much about it but I keep hearing about Snipe-IT, maybe that's closer to what you are looking for.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Andrei Morozov ***@***.***>
Sent: Monday, November 14, 2022 10:20 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Inventory spares management (Discussion #10911)


So I want to keep track of spare inventory items I have on my shelves in a quantifiable way - let's say for example

  *   70 units of "noname 40mm fan 4pin"
  *   35 units of "standard 1u PSU 450-600w"

something I absolutely have to have counted but do not care enough to store dozens of proper instances with their own serial numbers, roles, statuses and everything. Things just lying in boxes so my guys could routinely use them to swap similar ones when it's due. Of course changing the total number when they take one out or restock.

Any ideas on how to do that properly within Netbox? Am I trying to force on it something it's not meant for?

Google gives me 2 inventory plugins but they seem to solve tasks different from mine.
